doc/how-to-run-in-windows.rst

```diff
@@ -0,0 +1,352 @@
+Haskell, GHC and Cabal in Windows
+=================================
+
+The Windows toolchain and environment
+-------------------------------------
+
+`MinGW <https://www.mingw-w64.org/>`_ is a set of software packages for the
+Windows platform that enable development and compilation targeting the native
+Windows API. MinGW does not offer a complete developer environment so it usually
+is coupled with `MSYS2 <https://www.msys2.org/>`_ which provides many UNIX-like
+tools such as ``bash``, ``make`` or the package manager ``pacman``.
+
+MSYS2 comes with different environments. You can learn more about the different
+environments in the `MSYS2 documentation
+<https://www.msys2.org/docs/environments/>`_. We will be using ``UCRT64`` in the
+examples below but it can be swapped by ``CLANG64`` (as long as the external
+packages you want to use are available for the ``CLANG64`` environment, see
+:ref:`Using system libraries with GHC`).
+
+.. note::
+
+    When the code snippets must be run in a MSYS2 shell, we will use the ``msys2>``
+    prompt and when in a native Windows shell we will use the ``PS>`` or ``cmd>``
+    prompt. If either shell will do, we will just use ``>``.
+
+A small note on configuring and updating MSYS2

```

Comment:
I find the order here confusing. We start with notes about updating msys2, but there was no information on how to install it.

Reply:
I would have expected the user to click on the links above this part, read a bit about what is MinGW and MSYS2 and install MSYS2. It is true that I don't say that explicitly. Anyways, this section is gone now.